pom.xml

```diff
@@ -12,6 +12,7 @@
     <properties>
         <java.version>11</java.version>
         <vaadin.version>23.2-SNAPSHOT</vaadin.version>
+        <flow.version>2.7.15</flow.version>
```

Comment:
Flow version 2.7 for Vaadin 23.2 ?

Reply:
Ah, sorry, that was a rebase mistake. Actually the timing issue observed only with V14.8. Probably we miss some commit it TestBench